### ISSUE TITLE
BUG column (possibly game features)

### ISSUE THREAD

**Issue by gengyoubo** [LtxProgrammer/Changed-Minecraft-Mod#897]
- [ ]  Details of the latex Cats

- [ ]  Some cats can't trigger 1.

- [x]  dark latex companion can not save when injured

- [ ]  light latex will not alert the dark latex wolves

- [x]  White glue won't be harmed by black glue pools

- [ ]  dark latex that set off fireworks are not detected

- [ ]  Entering the dark latex will not aestivate it

- [x] Shields can't stop latex arrows

Quote this video：https://www.bilibili.com/video/BV1CZ421E79D/?spm_id_from=333.880.my_history.page.click&vd_source=686394728da821f4c2d652bbab263b5b

- [ ] Frequent terrain structures.
- [ ] Initial latex is too powerful, resulting in an inability to survive properly.
- [ ] Buildings generated without latex(#707 )
- [ ] Buildings should be rich in content and have few interactables, cans are inedible, etc.
- [ ] AI for latex could be smarter?
Quote to one of mcmod's comments：“I'm a returning player, and since I haven't played in too long, plus I saw this furry integrated pack so I tried it out and gave some of my personal advice:
First of all, the initial period, this latext mod, seems to be more frequent generation of the terrain structure, as well as will produce a fairly high quality of the monster, simply let me suffer, basically encountered on the death, the generation of the laboratory structure does not have any tips, I basically is the violence to solve only to find that there is a red stone organs, but there will be someone with red stone honestly decrypted? Laboratory into the not many monsters, the things inside are also more monotonous and pathetic, canned goods can not be eaten, interactivity is also less, I hope the author can continue to improve, the last of this integration pack is a gum beast.
      Finally, this package is a latex package, but the mission system is just a sketchy background story, I think it can be integrated into these structures to enrich the background storyline, as well as the current only able to turn into a latex and the transformation of some of the creatures, there is no vampire as much as the design of the latexes part of a kind of intelligent and also has a level of differentiation, you can consider further adding more content! 
, I hope the author can take part of my suggestions to do better and better.”

**Comment by LtxProgrammer** [comment]
Could you elaborate on what the first two entries are?

**Comment by gengyoubo** [comment]
> Could you elaborate on what the first two entries are?

From 0:20 to 2:30 of the video, the author shows 9 types of latex cats, which will explode if they get too close to the creeper (this is the first one). The last two however have no latex cat traits (this is the second one).

**Comment by gengyoubo** [comment]
- [ ] Frequent terrain structures.
- [ ] Initial latex is too powerful, resulting in an inability to survive properly.
- [ ] Buildings generated without latex(#707 )
- [ ] Buildings should be rich in content and have few interactables, cans are inedible, etc.
Quote to one of mcmod's comments：“I'm a returning player, and since I haven't played in too long, plus I saw this furry integrated pack so I tried it out and gave some of my personal advice:
First of all, the initial period, this gum beast mod, seems to be more frequent generation of the terrain structure, as well as will produce a fairly high quality of the monster, simply let me suffer, basically encountered on the death, the generation of the laboratory structure does not have any tips, I basically is the violence to solve only to find that there is a red stone organs, but there will be someone with red stone honestly decrypted? Laboratory into the not many monsters, the things inside are also more monotonous and pathetic, canned goods can not be eaten, interactivity is also less, I hope the author can continue to improve, the last of this integration pack is a gum beast.
      Finally, this package is a gum beast-based package, but the mission system is just a sketchy background story, I think it can be integrated into these structures to enrich the background storyline, as well as the current only able to turn into a gum beast and the transformation of some of the creatures, there is no vampire as much as the design of the gum beasts part of a kind of intelligent and also has a level of differentiation, you can consider further adding more content! 
, I hope the author can take part of my suggestions to do better and better.”
